FMSim/FMBase/Clubs/FootballClub.cs

```diff
@@ -0,0 +1,7 @@
+namespace FMBase.Clubs;
+
+public class FootballClub(string name, string description, string colors, int[] members, string formation, string jersey) : Club(name, description, colors, members)
+{
+    public string Formation = formation;
```

Comment:
Formation already implemented in Trainer class.

Reply:
Formation in Trainer is problem not good, make it in Club and fine, not my problem go ask Jan G.

Reply:
Implementation in Trainer was already confirmed, please remove this field from FootballClass.